client/imports/app/home/home.component.ts

```diff
@@ -6,4 +6,8 @@ import { Component, ChangeDetectionStrategy } from '@angular/core'
     styleUrls: [ './home.scss' ],
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class HomeComponent { }
+export class HomeComponent {
+    onSubmit() {
+        window.location.href = '/new-forum'
+    }
```

Comment:
I find the method name really unclear.

Reply:
And this shouldn't be on the home page publicly. Only admin users should be able to create forums.